daccustodian/paycpu.cpp

```diff
@@ -0,0 +1,24 @@
+
+void daccustodian::paycpu(name dac_id){
```

Comment:
From reading the code I think I understand what this is doing but it would be good to add some documentation to the README or via the ricardian contract line for this action to explain it a bit.